### ISSUE TITLE
Bumping adventure/minimessage broke <reset> in messages.yml

### ISSUE THREAD

**Issue by andrewkm** [DiscordSRV/DiscordSRV#1399]
### Expected behavior

<reset> tags to correctly reset the color.

### Actual behavior

Plugin built upon: https://github.com/DiscordSRV/DiscordSRV/commit/cc063da47fc0fda7b4d736bf379f61348026d858
Messages.yml: `DiscordToMinecraftChatMessageFormat: "[%channelcolor%Discord <white>|</white> %toprolecolor%%toprolealias%<reset>] %toprolecolor%%name%<reset>%reply% » %channelcolor%%message%"`

**Correctly** generates:
https://i.imgur.com/YO5QxF9.png

Plugin built upon: https://github.com/DiscordSRV/DiscordSRV/commit/58ab62b05ac337ae9c3485f17495907e5b1eb6ff
Messages.yml: `DiscordToMinecraftChatMessageFormat: "[%channelcolor%Discord <white>|</white> %toprolecolor%%toprolealias%<reset>] %toprolecolor%%name%<reset>%reply% » %channelcolor%%message%"`

**Incorrectly** generates:
https://i.imgur.com/17QjU4o.png

TL;DR formatting is all broken now.

### Steps to reproduce

See above.

### Server software and version

Paper 317 (1.18.2)

### Checks

- [X] I am __not__ using an outdated version of DiscordSRV.
- [X] I asked in [DiscordSRV's Discord server](https://discord.discordsrv.com) to see whether this issue is in fact a bug that needs to be fixed.
